src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -116,6 +124,15 @@ class adminLogs extends viewModelBase {
                 }
             }
         });
+        
+        this.logModeText = ko.pureComputed(() => {
```

Comment:
I think we don't need this observable at all.

Reply:
removed